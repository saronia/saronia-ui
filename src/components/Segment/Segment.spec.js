/* @flow */
import React from 'react';
import renderer from 'react-test-renderer';
import { matcher, serializer } from 'jest-styled-components';

import { Segment } from '.';

expect.addSnapshotSerializer(serializer);
expect.extend(matcher);

describe('<Segment />', () => {
  const tree = renderer.create(<Segment>Hello</Segment>).toJSON();

  it('renders a segment w/o crashing', () => {
    expect(tree.type).toEqual('div');
  });

  it('matches styled component snapshot', () => {
    expect(tree).toMatchStyledComponentsSnapshot();
  });
});
