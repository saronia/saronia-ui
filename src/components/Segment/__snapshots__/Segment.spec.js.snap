// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Segment /> matches styled component snapshot 1`] = `
.cRKSpU {
  position: relative;
  background: #FFFFFF;
  color: #1B1B1B;
  box-shadow: 0px 1px 2px 0 rgba(34, 36, 38, 0.15);
  padding: ;
  border-radius: 3px;
  border: 1px solid rgba(34, 36, 38, 0.15);
}





<div
  className="cRKSpU"
>
  Hello
</div>
`;
